attackers/response_handlers.py

```diff
@@ -0,0 +1,23 @@
+from PIL import Image
+import datetime
+import os
+
+
+def string_handler(params):
+    print(f"Server response:\n\t{params[0]}")
+    return params[1:]
+
+
+def image_handler(params: Image.Image):
+    time = datetime.datetime.now().strftime("%Y-%m-%d %H-%M-%S")
+    desktop = os.path.join(os.path.expanduser("~/OneDrive"), "Desktop")

```

Comment:
Who tells you this directory exists.  
1. It might not exists
2. You might not have permissions for that.
3. It might not run on windows.
4. ....